docs/versionhistory.rst

```diff
@@ -40,6 +40,8 @@ This library adheres to `Semantic Versioning 2.0 <http://semver.org/>`_.
   * ``anyio.Path.with_segments()``
 
   (PR by Ganden Schaffner)
+- Added support for the python 3.12 ``walk_up`` keyword argument in
+  ``anyio.Path.relative_to()`` (PR by Colin Taylor)
```

Comment:
Don't try the amend the release notes of an already published release. Add a new section for **UNRELEASED**.

Reply:
Apologies, wasn't intended - I transposed into the wrong section. I'll shift to the correct place